Lombiq.HelpfulLibraries.OrchardCore/DependencyInjection/InlineStartup.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Modules;
+using System;
+
+namespace Lombiq.HelpfulLibraries.OrchardCore.DependencyInjection;
+
+/// <summary>
+/// A startup class that invokes the delegates provided in the constructor.
+/// </summary>
+public class InlineStartup : StartupBase
```

Comment:
I just learned about [`IStartupFilter`](https://andrewlock.net/exploring-istartupfilter-in-asp-net-core/) ([docs](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/startup?view=aspnetcore-8.0)). Seems like the same thing?

Reply:
As I understand `IStartupFilter` does more of less the same as `OrchardCore.Modules.IStartup`, just not OC specific. I don't think this is helpful here, because I'd rather keep the assumption that OC is initialized by the time this startup is activated.

Reply:
Good point, OK then.